tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.SharePointAppWeb/App_GlobalResources/Constants.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Central_Repository_Url" xml:space="preserve">
-    <value>[Enter URL of the central repository where all the common lists are stored, e.g. https://[TENANTNAME].sharepoint.com/sites/catalog]</value>
+    <value>https://MyTenant.sharepoint.com/sites/catalog</value>

```

Comment:
If we have the URL like this, there are chances that, user may think that URL has already been provided and he may not update the correct url. If we have placeholder like the old ones, the user will update the value